helm/prometheus-meta-operator/templates/ingress-oauth.yaml

```diff
@@ -18,7 +18,17 @@ spec:
           serviceName: oauth2-proxy
           servicePort: 4180
         path: /oauth2
+  - host: {{ .Values.Installation.V1.Monitoring.Alertmanager.Host }}
+    http:
+      paths:
+      - backend:
+          serviceName: oauth2-proxy
+          servicePort: 4180
+        path: /oauth2
```

Comment:
Just out of curiosity, why add this here instead of keeping everything AlertManager related in `alertmanager-ingress.yaml`?

Reply:
Because they do not share the same annotations :)

Reply:
Well, but if this is essentially moving from https://github.com/giantswarm/g8s-prometheus/blob/master/helm/g8s-prometheus/templates/alertmanager/ingress.yaml then it does have the `tls-acme` annotation.

I don't know, I'll leave this up to you to decide, the reason I asked is that the `ingress-oauth.yaml` already conflates settings for Prometheuses and Promxy just because they happen to share a domain. It breaks the single-responsibility principle and means that e.g. Promxy cannot be deployed without PMO.

But since we're moving AlertManager into PMO anyway, then it doesn't matter anyway.